open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-client/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/client/util/Queries.java

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.governanceservers.openlineage.client.util;
+
+public enum Queries {
+
+    glossary("glossary"),
```

Comment:
same here.
Please use uppercase letters.

Reply:
Hi Bogdan, 
I made a conscious decision here to deviate from the naming convention because these enum values are used to distinguish between the GET parameters of my endpoint. Having uppercase GET values would be even less ideal I think. See also the GraphServices class. Shall I leave it like this, or would you have a different idea on how to deal with this problem?

Reply:
Uppercase value for http parameters is not an issue.